.github/workflows/static_analysis.yml

```diff
@@ -0,0 +1,44 @@
+name: Static Analysis
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  coverage:
+    name: SonarQube - (master)
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+      - name: Checkout repo (master)
+        uses: actions/checkout@v2
+        with:
+          # For sonar-scanner to work properly we can't use a shallow fetch.
+          fetch-depth: 0
+
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install pipenv
+        uses: dschep/install-pipenv-action@v1
+
+      - name: Install Remaining Dependencies
+        run: pipenv install --dev --system
+
+      - name: Run unit tests with coverage
```

Comment:
Can you tell me why we are running tests twice?

Reply:
Sonarcloud doesn't make the report we've run for the PR apply to master on merge - so we have to re-run it against the master branch after merging.